tasks/src/Zero-Shot_Image_Classification/data.ts

```diff
@@ -0,0 +1,30 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "",
+			id:          "",
+		},
+		{
+			description: "",
+			id:          "",
+		},
+	],
+	
+	models: [
+		{
+			description: "",
+			id:          "",
+		},
+		{
+			description: "",
+			id:          "",
+		},
+	],
+	summary:      "Zero-shot image classification is a task in computer vision where the aim is to classify images into one of several pre-defined categories, without any prior training or knowledge of the categories.",

```

Comment:
It's good to keep PRs minimal for the sake of review process, so I'd encourage you to remove the files that aren't about `zero-shot-classification` (also note that we already have someone contributed for this and given this is PR for zero-shot-classification)